src/operation.js

```diff
@@ -0,0 +1,62 @@
+/** @format */
+
+import { getFromLocalStorage, setLocalStorage } from './local_storage.js';
+
+const input = document.querySelector('.input');
+const tasksList = document.querySelector('.list');
+export default class Todo {
+  tasks;
+
+  constructor() {
+    this.tasks = getFromLocalStorage();
+  }
+
+  addTask = (e) => {
+    e.preventDefault();
+    const task = {
+      index: this.tasks.length,
+      description: input.value,
+      completed: false,
+    };
+    this.tasks.push(task);
+    setLocalStorage(this.tasks);
+    this.displayTasks();
+  };
```

Comment:
- [OPTIONAL] I suggest clearing the input after you add a new task by adding this line `input.value = ' '; ` 💯

Reply:
Thank you @Bondok6 for your feedback, I will Modify It.